src/thirdparty/is-animated/package.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
remove this file

Reply:
@gideonthomas Thank you for the comments, I'll remove the part and files!